frontend/src/LabSection.js

```diff
@@ -8,7 +8,7 @@ import Checkbox from '@material-ui/core/Checkbox';
 import IconButton from '@material-ui/core/IconButton';
 import DeleteIcon from '@material-ui/icons/Delete';
 
-class Section extends React.Component {
+class LabSection extends React.Component {
```

Comment:
Agreed that we should avoid creating a duplicate class for lab sections. Unfortunately, I don't know enough about React to recommend the best way to avoid that - maybe Fabrice can suggest a solution when he's back tomorrow.